### PR TITLE
Remove redundant parenthesis for ARMv7l build.

### PR DESCRIPTION
**Pull request by tellison**

### CODE REVIEW

File: makejdk-any-platform.sh [hunk 1]
```diff
@@ -41,7 +41,7 @@ fi
 if [[ "$OS_MACHINE_NAME" == "armv7l" ]] ; then
    export JVM_VARIANT=${JVM_VARIANT:-zero}
    export MAKE_ARGS_FOR_ANY_PLATFORM=${MAKE_ARGS_FOR_ANY_PLATFORM:-"DEBUG_BINARIES=true images"}
-   export CONFIGURE_ARGS_FOR_ANY_PLATFORM=${CONFIGURE_ARGS_FOR_ANY_PLATFORM:-"--with-jobs=${NUM_PROCESSORS}}"}
```
Comment (neomatrix369) [inline makejdk-any-platform.sh]: Good catch @tellison

Comment (tellison) [inline makejdk-any-platform.sh]: Thanks, but I can't take credit for that.  It was Mr Jenkins who found it.
I guess nobody wondered/cared why the ARM build was failing ;-( !?

Comment (neomatrix369) [inline makejdk-any-platform.sh]: Can I be pls added to the jenkins build notification messages, especially when it fails.

Comment (neomatrix369) [inline makejdk-any-platform.sh]: There is an email notification action on the jenkins jobs, should we add a email list to it and have ourselves subscribe to it? Or is there something in place already for it?

**Pull request merged**